pkg/flag/flag.go

```diff
@@ -28,23 +28,34 @@ const (
 	TriggerControllerName = "tekton-triggers-controller"
 	TriggerWebhookName    = "tekton-triggers-webhook"
 
-	AnnotationPreserveNS = "operator.tekton.dev/preserve-namespace"
+	AnnotationPreserveNS      = "operator.tekton.dev/preserve-namespace"
+	LabelTaskProviderType     = "operator.tekton.dev/provider-type"
```

Comment:
`ProviderTypeLabel` may read better.

Reply:
individually yes,
but when the labels and annotations are grouped in adjacent lines, the `<TYPE><NAME>` looks better.

In addition, it follows custom error definitions is go. `ERROR<TYPE/MESSAGE>`